spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      expect(other_user).to receive(:lock_access!)
+      allow(User).to receive(:find).and_return(other_user)
+
+      post other_user.expect(other_user.reload.access_locked?).to be_truthy
```

Comment:
@jhass 
This line: 
> other_user.expect(other_user.reload.access_locked?).to be_truthy

returns the error "undefined method `expect' " 
Is this the expected behavior? Kindly explain.